packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx

```diff
@@ -208,14 +212,15 @@ export default class RoosterCommandBar extends React.PureComponent<RoosterComman
             return null;
         }
 
-        const { strings, calloutClassName, calloutOnDismiss } = this.props;
+        const { strings, calloutClassName, calloutOnDismiss, tooltipDirectionHint } = this.props;
         const className = commandBarButton.className || "";
         const rootClassName = className.split(" ").indexOf(RoosterCommandBarButtonRootClassName) < 0 ? RoosterCommandBarButtonRootClassName : undefined;
         // make a copy of the OOB button template since we're changing its properties
         const button = { ...commandBarButton, className: css(rootClassName, { "first-level": firstLevel }, className) };
 
-        if (!button.onRender && button.onRenderParams) {
-            button.onRender = getIconOnRenderDelegateWithCustomCacheKey(button.key + this._id, ...button.onRenderParams);
+        if (!button.onRender && button.onRenderOptions) {
+            const { customCacheKey = `${button.key}${this._id}` } = button.onRenderOptions;
```

Comment:
I'm a little confused on what is happening in this line. What does customCacheKey and button.onRenderOptions become?

Reply:
If customCacheKey (a member of button.onRenderOptions) isn't specified (i.e. it's undefined), we specify a default one. Does that answer your question?

Reply:
oh okay i see. yes that makes sense. thanks!